Client/Cards/actions/fake.spec.ts

```diff
@@ -0,0 +1,41 @@
+import * as dotenv from "dotenv"
+import * as pax2pay from "../../../index"
+import { ErrorResponse } from "../../../model"
+import { actionTest } from "./actionTest"
+import { factory } from "./factory"
+
+dotenv.config()
+jest.setTimeout(200000)
+
+describe("pax2pay.cards.actions fake", () => {
+	const client = pax2pay.Client.create(process.env.url)
+	beforeAll(
+		async () =>
+			await client?.auth.login({
+				username: process.env.username ?? "user",
+				password: process.env.password ?? "password",
+			})
+	)
+	for (const currency of ["EUR", "GBP", "SEK", "JPY", "BRL"])
+		for (const cardType of ["VISA_CREDIT", "MASTERCARD"])
```

Comment:
i don't think we need to do one for each, that seems a bit excessive. just one card should do here, we're testing the state changes not the card creations and types